org.eclipse.winery.workflowmodeler/src/app/model/workflow.node.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2017 ZTE Corporation.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and the Apache License 2.0 which both accompany this distribution,
+ * and are available at http://www.eclipse.org/legal/epl-v10.html
+ * and http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Contributors:
+ *     ZTE - initial API and implementation and/or initial documentation
+ */
+
+export const WorkflowNodeType = [
+    'StartEvent',
+    'EndEvent',
+    'ToscaNodeManagementTask',
+    'ExclusiveGateway',
+    'ParallelGateway',
+];
+
+export class WorkflowNode {
+    public id: string;
+    public name: string;
+    public type: string;
+    public input: any[];
+    public output: any[];
+    public connection: any[];
+    public template: any;
+    public position: any;
+
+    public nodeInterface: string;
+    public nodeTemplate: string;
+    public nodeOperation: string;
+
+    constructor({

```

Comment:
Did some changes as you suggested. I created some class to instead of object. And changed the names of some properties for the names are not appriate for properties. For example, interface is a reserved word, I changed it to 'nodeInterface', and for 'node_template' underline is not suggested, so I changed it to 'nodeTemplate'.